### ISSUE TITLE
Project Structure Documentation page not working

### ISSUE THREAD

**Issue by aniketk13** [Tech-Phantoms/Tech-Phantoms#323]
### What would you like to share?

The Project Structure Documentation link in readme leads to a "Page not found". It should lead to a page with proper context which would help especially the beginners to understand and contribute to the project.

### Would you like to work on this issue?

- [ ] Yes, I want to work on this issue!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Souvikns** [comment]
Yeah, the markdown file is removed now so it is showing 404. I think we have two options here 
- we remove the link and update the readme and act as if was never there 
- we create a new project-structure file and talk about the new project structure since the project has moved to nextjs.
